### DISCUSSION TITLE
Add metrics to monitor Backend/CQLStoreManager ThreadPools

### DISCUSSION THREAD

**Discussion by cdegroc** [JanusGraph/janusgraph#2740] [Ideas]
Now that PR #2700 has been merged, we can tune the Backend and CQLStoreManager ExecutorServices.
In order to better monitor them, we could generate metrics using Dropwizard's `InstrumentedExecutorService`/`InstrumentedThreadFactory`.
Here is a first attempt at adding this: https://github.com/cdegroc/janusgraph/commit/589f560deb4bfb0d63390e3ee52d4d404466378d

I would like to gauge interest/feedback before opening a PR.

Thanks!

**Comment by li-boxuan** [comment]
I think that's a very nice feature to have.

**Reply by cdegroc** [reply]
Opened PR #2744 👍

**Comment by porunov** [comment]
This feature was implemented by @cdegroc in #2744 .
Closing this discussion as resolved.
